src/Client/scss/main.scss

```diff
@@ -171,6 +171,24 @@ body { width: 100%; }
 
         
     }
+
+    /* override Bulma's margin, we don't need it */
+    .columns:not(last-child) {
```

Comment:
Fulma's standard column control had a default margin which was not looking good in our scenario